### DISCUSSION TITLE
Can the synthesizer be tuned to a specific midi channel?

### DISCUSSION THREAD

**Discussion by Opekusha** [marcel-licence/esp32_basic_synth#30] [Q&A]
Can the synthesizer be tuned to a specific midi channel? So that he would not perceive other channels. The sequencers allow you to tune any of the 16 channels. This is very useful when there are several synthesizers. VST synthesizers are good, but more interesting for me than hardware.

**Comment by marcel-licence** [comment]
Moved this to Q&A.

Yes sure. Note on messages to provide the MIDI channel. So you can just drop messages you are not interested in "_NoteOn(uint8_t ch, uint8_t note, uint8_t vel)"

For example:
 if (ch != 1) { return; }.
